argo_rollouts/assets/dashboards/argo_rollouts_overview.json

```
@@ -0,0 +1,1589 @@
+                            "height": 1,
+                            "width": 6,
+                            "x": 0,
+                            "y": 0
+                        }
+                    },
+                    {
+                        "definition": {
+                            "check": "argo_rollouts.openmetrics.health",
+                            "group_by": [
+                                "host",
+                                "endpoint"
+                            ],
+                            "grouping": "cluster",
+                            "tags": [],
+                            "title": "Endpoint Health",
+                            "title_align": "center",
+                            "title_size": "16",
+                            "type": "check_status"
+                        },
+                        "id": 8408317728958814,
+                        "layout": {
+                            "height": 3,
+                            "width": 2,
+                            "x": 0,
+                            "y": 1
+                        }
+                    },
+                    {
+                        "definition": {
+                            "color_preference": "text",
+                            "count": 50,
+                            "display_format": "list",
+                            "hide_zero_counts": true,
+                            "query": "tag:integration:argo_rollouts",
+                            "show_last_triggered": false,
+                            "show_priority": false,
+                            "sort": "status,asc",
+                            "start": 0,
+                            "summary_type": "monitors",
+                            "title": "Argo Rollout Monitors",
+                            "type": "manage_status"
+                        },
+                        "id": 6941619982932726,
+                        "layout": {
+                            "height": 3,
+                            "width": 4,
+                            "x": 2,
+                            "y": 1
+                        }
+                    }
+                ]
+            },
+            "id": 5696881176276130,
+            "layout": {
+                "height": 5,
+                "width": 6,
+                "x": 6,
+                "y": 0
+            }
+        },
+        {
+            "definition": {
+                "background_color": "vivid_orange",
+                "layout_type": "ordered",
+                "show_title": true,
+                "title": "Rollout Overview",
+                "type": "group",
+                "widgets": [
+                    {
+                        "definition": {
+                            "background_color": "orange",
+                            "content": "The Rollout Overview offers a high-level overview of the progress of rollouts by displaying the current phases in which a rollout is currently.",
```

Non blocking recommendation
```suggestion
                            "content": "The Rollout Overview offers a high-level summary of the progress of rollouts by displaying the current phases in which a rollout is currently.",
```